src/pages/trading-specification/data/_crypto_specification.ts

```diff
@@ -29,6 +29,7 @@ const crypto_specification: TSpecification = {
         {
             instrument: {
                 symbol: 'ADAUSD',
+                swf_icon: true,
```

Comment:
shouldn't we add falsy statements as well?

Reply:
not required. swf_icon is conditional property.